tensorrt_llm/_torch/memory_buffers_util.py

```
@@ -0,0 +1,63 @@
+class Buffers:
+
+    def __init__(self):
+        buffers: {str, list[(torch.Tensor, bool)]} = {}
+
+    def get_buffer(self, tensor_shape: list[int], dtype: torch.dtype,
+                   buffer_name: str, create_if_miss: bool, pin_memory: bool):
+
+        dtype.itemsize
+
+        def select_buffer_with_more_elements(
+            pinned_buffer: Optional[torch.Tensor],
```

Ruff (0.12.2)

Undefined name `Optional`

(F821)